ios/NativeSigner/Components/HistoryCard.swift

```diff
@@ -64,6 +64,14 @@ struct HistoryCard: View {
                 line1: Localizable.HistoryCard.keysRemoved.string,
                 line2: value.seedName + value.path
             )
+        case let .secretWasExported(value):
+            HistoryCardTemplate(
+                image: .init(.eye, variants: [.trianglebadge, .exclamationmark, .fill]),
+                timestamp: timestamp,
+                danger: true,
+                line1: "Secret was exported",
```

Comment:
After including changes on `master`. please align it by using localizable.

1. Open `Localizable.strings`, add there `"HistoryCard.SecretWasExported" = "Secret was exported";`
2. Build the app to regenerate the `enum Localizable`
3. replace this line with Suggested code


```suggestion
                line1: "Localizable.HistoryCard.secretWasExported.string
```

Reply:
No blocker for merging, I'm happy to refactor that on any of my next PRs 🙏🏻

Reply:
Thanks for catching that! Fixed